packages/react-native/Libraries/Text/Text/RCTTextView.mm

```diff
@@ -16,6 +16,14 @@
 
 #import <QuartzCore/QuartzCore.h>
 
+#ifdef __IPHONE_16_0
```

Comment:
I don't think we need these ifdef's: we're always building with the iOS16 SDK or newer.

Reply:
Ah okay, I wasn't sure about that one. I will remove it